### ISSUE TITLE
Polyembolokoilamania

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4785]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
Insertion of foreign objects into body orifices

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
adding as HP_0032508  @MickeySegal
